ui/src/main/java/com/adyen/checkout/ui/internal/ButtonDefaults.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2025 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by oscars on 16/4/2025.
+ */
+
+package com.adyen.checkout.ui.internal
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import com.adyen.checkout.ui.theme.AdyenButtonStyle
+
+internal object ButtonDefaults {
+
+    @Composable
+    fun primaryButtonStyle(style: AdyenButtonStyle?): InternalButtonStyle {
+        val colors = AdyenCheckoutTheme.colors
+        return InternalButtonStyle(
+            backgroundColor = style?.backgroundColor?.toCompose() ?: colors.primary,
```

Comment:
Is there something explicitly mentioned in the design that we can refer about which color to use as fallback? If in the future we see an issue with some of these colors due to using fallback, then it would be good to have them documented somewhere.